### PR TITLE
Prevent editing wrong label when changing column

### PR DESCRIPTION
**Pull request by boutinb**
Fixes jasp-stats/jasp-test-release#715

### CODE REVIEW

**Comment by boutinb** [comment]
@JorisGoosen This fix is not really beautiful... do you have any suggestion?

File: JASP-Desktop/components/JASP/Widgets/VariablesWindow.qml [hunk 1]
```diff
@@ -27,20 +27,23 @@ FocusScope
...
 	Connections
 	{
 		target: labelModel
 		
 		onChosenColumnChanged:
 		{
+			blockEditing = true;
 			if(labelModel.chosenColumn > -1 && labelModel.chosenColumn < dataSetModel.columnCount())
 			{
 				//to prevent the editText in the labelcolumn to get stuck and overwrite the next columns data... We have to remove activeFocus from it
```
Comment (JorisGoosen) [inline JASP-Desktop/components/JASP/Widgets/VariablesWindow.qml:41]: Did you see this comment?

Comment (JorisGoosen) [reply JASP-Desktop/components/JASP/Widgets/VariablesWindow.qml:41]: Well, Ill try to see if I can fix it without an extra boolean

**Comment by JorisGoosen** [comment]
So, Ive made an alternative fix, it still needs an extra property sadly enough, but not one that we have to set to true and then later again to false. (If that goes wrong the whole thing stops working...)

See: https://github.com/JorisGoosen/jasp-desktop/commit/9438696f801aca9250538359ffddbf4c2b10453a

**Comment by JorisGoosen** [comment]
You get to pick which one is prettier :)

**Comment by JorisGoosen** [comment]
So I made a PR for you to make it easy: https://github.com/jasp-stats/jasp-desktop/pull/3991

**Pull request closed without merging**